starboard/linux/x64x11/system_get_property_impl.cc

```diff
@@ -32,11 +32,7 @@ const char kModelName[] = "ModelName";
 const char kPlatformName[] = "X11; Linux x86_64";
 const char kSystemIntegratorName[] = "SystemIntegratorName";
 
-#if SB_API_VERSION >= 14
 const char kModelYear[] = "2023";
```

Comment:
This should probably be updated to `2024`. See `c4675c27c2bd96df3b1ed2f564cb74f001ecba60`

Reply:
Fixed to reduce boilerplate churn for next sb version.